nss/src/lib.rs

```diff
@@ -31,3 +30,29 @@ lazy_static! {
 fn init_logger() {
     logs::init_logger();
 }
+
+/// socket_path returns the socket path to connect to the gRPC server.
+///
+/// It uses the NSS_AUTHD_SOCKET env value if set and the custom_socket feature is enabled,
+/// otherwise it uses the default path.
+fn socket_path() -> String {
+    #[cfg(feature = "custom_socket")]
+    match std::env::var("NSS_AUTHD_SOCKET") {
```

Comment:
I suggest we start the env variable by AUTHD so that everything authd related is properly namespaced.